eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelay.scala

```diff
@@ -191,32 +191,37 @@ class ChannelRelay private(nodeParams: NodeParams,
   private val nextNodeId_opt = channels.headOption.map(_._2.nextNodeId)
 
   /**
-   * Select a channel to the same node to relay the payment to, that has the lowest balance and is compatible in
-   * terms of fees, expiry_delta, etc.
+   * Select a channel to the same node to relay the payment to, that has the lowest capacity and balance and is
+   * compatible in terms of fees, expiry_delta, etc.
    *
    * If no suitable channel is found we default to the originally requested channel.
    */
   def selectPreferredChannel(alreadyTried: Seq[ShortChannelId]): Option[ShortChannelId] = {
     val requestedShortChannelId = r.payload.outgoingChannelId
     context.log.debug("selecting next channel")
     nextNodeId_opt match {
-      case Some(nextNodeId) =>
+      case Some(_) =>
         // we then filter out channels that we have already tried
         val candidateChannels: Map[ShortChannelId, OutgoingChannel] = channels -- alreadyTried
         // and we filter again to keep the ones that are compatible with this payment (mainly fees, expiry delta)
         candidateChannels
           .map { case (shortChannelId, channelInfo) =>
             val relayResult = relayOrFail(Some(channelInfo.channelUpdate))
-            context.log.debug(s"candidate channel: shortChannelId={} balanceMsat={} channelUpdate={} relayResult={}", shortChannelId, channelInfo.commitments.availableBalanceForSend, channelInfo.channelUpdate, relayResult)
+            context.log.debug(s"candidate channel: shortChannelId={} capacitySat={} balanceMsat={} channelUpdate={} relayResult={}", shortChannelId, channelInfo.commitments.commitInput.txOut.amount, channelInfo.commitments.availableBalanceForSend, channelInfo.channelUpdate, relayResult)
```

Comment:
Nit, feels more natural that way:

```suggestion
            context.log.debug(s"candidate channel: shortChannelId={} balanceMsat={} capacitySat={} channelUpdate={} relayResult={}", shortChannelId, channelInfo.commitments.availableBalanceForSend, channelInfo.commitments.commitInput.txOut.amount, channelInfo.channelUpdate, relayResult)
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1539/commits/8c0876d6d2c568851a2be27ac0bd9f77597f5908